packages/connect-core/src/callback-client.ts

```diff
@@ -124,7 +124,7 @@ function createUnaryFn<I extends Message<I>, O extends Message<O>>(
  * ServerStreamingFn is the method signature for a server-streaming method of
  * a CallbackClient.
  */
-type ServerStreamingFn<I extends Message, O extends Message> = (
+type ServerStreamingFn<I extends Message<I>, O extends Message<O>> = (
```

Comment:
@timostamm We should probably mention this in the release notes for protobuf-es, yeah?